java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/BaseApiCaller.java

```diff
@@ -61,7 +61,7 @@
 
 class BaseApiCaller {
 
-    private static final Logger logger = LoggerFactory.getLogger(BaseApiCaller.class);
+    protected static final Logger logger = LoggerFactory.getLogger(BaseApiCaller.class);
```

Comment:
I think it's much better for each class to have their own Logger instance. This way, if we get a log message from BaseApiCaller, we can know for sure it comes from BaseApiCaller instead of having to hunt through all of its subclasses to find out where the log message is coming from.